modules/microshift-multus-intro.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-intro_{context}"]
+= Additional networks in {microshift-short}
+
+During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the Multus CNI plugin, you can add additional interfaces to pods from other networks. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
+
+[id="additional-network-use-cases"]
+== Use case for an additional network
```

Comment:
I would suggest a heading that describes the use case--something like "Using additional networks for network isolation" or similar.